tests/test_buffers.py

```diff
@@ -62,6 +62,8 @@ def test_to_python():
     assert cstats.alive() == 1
     del mat2  # holds a mat reference
     pytest.gc_collect()
+    pytest.gc_collect()  # Needed for PyPy
```

Comment:
Can these be removed again? (Looks like a last-ditch brute force attempt.)

Reply:
I think it didn't work, PyPy still didn't like this. So yes.